packages/angular_devkit/build_angular/package.json

```diff
@@ -27,6 +27,7 @@
     "babel-plugin-istanbul": "6.1.1",
     "browserslist": "^4.9.1",
     "cacache": "16.1.3",
+    "chokidar": "3.5.3",
```

Comment:
Can you please add this to the main package.json as well?

Reply:
Already there but not pinned (i'll fix that).
https://github.com/angular/angular-cli/blob/6f8142ec7e5e9f8b074411d8ff2efa8e6bbad979/package.json#L139